feat(apigatewayv2): websocket api (#13031)
aws/aws-cdk: packages/@aws-cdk/aws-apigatewayv2-integrations/README.md

  }),
});
```

## WebSocket APIs

> WebSocket integrations connect a route to backend resources. Currently supported integrations include: LambdaWebSocketIntegration

Reviewer: Just so we don't have to constantly update this list

```suggestion
WebSocket integrations connect a route to backend resources. The following integrations are supported in the CDK.
```

### Lambda WebSocket Integration

> Lambda integrations enable integrating a WebSocket API route with a Lambda function. When a client connects/disconnects or sends message specific to a route, the API Gateway service forwards the request to the Lambda function

Reviewer: Try to keep line width to 120 or at most 150

```suggestion
Lambda integrations enable integrating a WebSocket API route with a Lambda function. When a client connects/disconnects
or sends message specific to a route, the API Gateway service forwards the request to the Lambda function.
```

> The API Gateway service will invoke the lambda function with an event payload of a specific format.

Reviewer: Any details we can link to on what the lambda event request will contain and what the schema of the response should be.

Author: I looked around the API reference and developer docs but was not able to find anything unfortunately. We can add it in later changes once we have one.
